traffic_ops/app/lib/UI/ConfigFiles.pm

```diff
@@ -533,9 +550,8 @@ sub ip_allow_data {
 		}
 
 		# compact, coalesce and compact combined list again
-		# if more than 5 servers are in a /24, list that /24 - TODO JvD: parameterize
 		my @compacted_list = NetAddr::IP::Compact(@allowed_netaddrips);
-		my $coalesced_list = NetAddr::IP::Coalesce( 24, 5, @allowed_netaddrips );
+		my $coalesced_list = NetAddr::IP::Coalesce( $coalesce_masklen_v4 , $coalesce_masklen_v4, @allowed_netaddrips );
```

Comment:
I think you meant

my $coalesced_list = NetAddr::IP::Coalesce( $coalesce_masklen_v4 , $coalesce_number_v4, @allowed_netaddrips );